tests/github/test_rate_limiter.py

```
@@ -16,6 +16,34 @@
 
 import time
 
+from tests.conftest import rate_limiter
```

Ruff (0.11.9)

`tests.conftest.rate_limiter` imported but unused

Remove unused import: `tests.conftest.rate_limiter`

(F401)